src/Microsoft.TemplateEngine.Cli/LocalizableStrings.resx

```diff
@@ -276,6 +276,12 @@
   <data name="NoTemplatesMatchName" xml:space="preserve">
     <value>No templates matched the input template name: {0}.</value>
   </data>
+  <data name="OptionalWorkloadsSynchronized" xml:space="preserve">
+    <value>The list of templates was synchronized with the Optional SDK Workloads.</value>
```

Comment:
Did we get Kathleen's approval on the localizations or not yet?

Reply:
Not yet discussed.
It's not blocking us, we can fix this later.